dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/resources/docker/file-manage/common.properties

```diff
@@ -0,0 +1,90 @@
+#
```

Comment:
If the original file has changed, does it need to be kept in sync here?

Reply:
It only need to when the test covers the modified function.